workspaceThumbnail.js

```diff
@@ -231,4 +234,29 @@ var ThumbnailsBoxOverride = {
         childBox.x2 += indicatorRightFullBorder;
         this._indicator.allocate(childBox);
     }
-}
\ No newline at end of file
+}
+
+var WorkspaceThumbnailOverride = {
+    after__init: function() {
+        this._contents.add_actor(this._getBackground().backgroundActor);
+    },
+
+    _getBackground: function() {
+        if (!this._bgManager)
+            this._bgManager = new Background.BackgroundManager({ monitorIndex: Main.layoutManager.primaryIndex,
+                                                             container: this._contents,
+                                                             vignette: false });
+        return this._bgManager;
+    },
+
+    _onDestroy: function() {
+        this.workspaceRemoved();
+
+        if (this._bgManager) {
+            this._bgManager.destroy();
+            this._bgManager = null;
+        }
+
+        this._windows = [];
+    },
```

Comment:
- The Background manager already takes care of adding a background actor, adding it manually will cause errors (also add_actor is deprecated, it should be add_child now).
- Overwriting ``_onDestroy`` does not work since it is bound in ``WorkspaceThumbnail``'s init so we'll have to add an extra callback on the signal.

I think we can replace the whole thing with:
```suggestion
    after__init: function () {
        this._bgManager = new Background.BackgroundManager({
            monitorIndex: Main.layoutManager.primaryIndex,
            container: this._contents,
            vignette: false
        });

        this.connect('destroy', (function () {
            this._bgManager.destroy();
            this._bgManager = null;
        }).bind(this));
    }
```